build/xunit.runsettings

```diff
@@ -1,7 +1,10 @@
 <RunSettings>
```

Comment:
Do we still need this?

Reply:
I think so, its being passed via the command-line to some commands:

https://github.com/NuGet/NuGet.Client/pull/4497/files#diff-22d0028b46aaa349a79daea8ecb058bec0d2b1a6b262bce683c99e5d62d1696cR46

Reply:
Off topic question: @jeffkl  how did you select line from PR diff?

Reply:
You can click on the line numbers in the diff to get a link to that part.

Reply:
Got it.

Reply:
Ok, so looks like these settings are not xunit.runner only, they'd be respect by dotnet test as well.

Reply:
Yes technically we’re still using the xunit “runner” but it runs under vstest.console instead of xunit.console.